quotes/tests.py

```diff
@@ -170,3 +197,191 @@ def test_vote_up_pending_fails(self):
         self.client.login(username='test', password='password')
         response = self.client.post(reverse(vote_up, kwargs={'quote_id': 1}))
         self.assertEqual(response.status_code, 404)
+
+    def test_vote_up_GET_405s(self):
```

Comment:
It is not immediately clear what this test wishes to accomplish, wrt. the other vote up tests. Can we rename it to make it more clear?

Reply:
It tests that sending a GET request to the vote_up endpoint results in a 405 (Method not allowed). I feel it is quite clear. But I can increase the verbosity somewhat if that helps.